styles.css

```diff
@@ -103,9 +103,20 @@ table {
   border-collapse: collapse;
 }
 
-th {
+th button {
```

Comment:
The 'Topics' header doesn't match the buttons this way. Making this `th, th button` fixes it, though.

Reply:
But then the other headers changes their cursor and so on.
It would be best if they all had the same basic styling, eg.
```
th {
 // base styles for all headers
}
```
and the buttons would have something extra
```
th button {
 // only styles related to the header with button
 // eg. cursor, transition...
}
```
Remember that it is CSS so the properties can be overridden or extended...

Reply:
Yeah, definitely keep some styles separate for the buttons, but make sure to include the buttons in the first selector, too. That way, they'll still be affected. Buttons have their own base styling that gets in the way if it's not overridden. So...

```
th, th button {
// base header styles
}
```

and then

```
th button {
// styles specific to buttons (like cursor)
}
```

Reply:
I just noticed that topic header looks different than other header. I remember initially it all looked the same.

Reply:
Last thing I see is to move the `cursor: pointer` statement into `th button:hover`. As it is, the 'Topics' header looks like a link, too, but it doesn't do anything. Just moving that statement should fix it, though. Everything else looks great.

Reply:
Try this...

```
thead {
    border-bottom: 1px solid var(--primary-color);
    line-height: 60px; /* <= remove this line */
}

th, th button {
    font-size: 22px;
    font-weight: 300;
}

th button {
  font-family: inherit;
  background-color: inherit;
  color: inherit;
  padding: 0.25em 0.75em;
  border: none;
  cursor: pointer;
  transition: 0.5s background-color;
}
```

Reply:
Yeah, that looks better. Either way works, but @Dorfieeee 's solution keeps it more consistent with the original.